scene/animate-images-with-image-overlay/src/main/java/com/esri/samples/animate_images_with_image_overlay/AnimateImagesWithImageOverlayController.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright 2020 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.animate_images_with_image_overlay;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+
+import com.esri.arcgisruntime.geometry.Envelope;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.layers.ArcGISTiledLayer;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.ImageFrame;
+import com.esri.arcgisruntime.mapping.view.ImageOverlay;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.Slider;
+
+public class AnimateImagesWithImageOverlayController {
+
+  @FXML
+  private SceneView sceneView;
+  @FXML
+  private Button controlAnimationButton;
+  @FXML
+  private Slider opacitySlider;
+  @FXML
+  private ComboBox<String> framesComboBox;
+
+  private List<ImageFrame> imageFrames;
+
+  private Integer imageIndex = 0;
```

Comment:
It might also make more sense to call it 'frameIndex', since we use it to pick an entry from `List<ImageFrame> imageFrames`, not from `imageFiles[]`

Reply:
this has to be a member variable so that the count is tracked correctly: instantiating it as 0 inside the variable just makes the index set to 0 everytime. If I'm misunderstanding, could you let me know what you were thinking please?

Reply:
I got mixed up, I meant perhaps instantiate it in the `initialize` method, but really it makes sense where it is just now, so lets leave it as is.